tests/pyfunc/docker/conftest.py

```diff
@@ -64,6 +64,9 @@ def save_model_with_latest_mlflow_version(flavor, extra_pip_requirements=None, *
     latest_mlflow_version = get_released_mlflow_version()
     if flavor == "langchain":
         kwargs["pip_requirements"] = [f"mlflow[gateway]=={latest_mlflow_version}", "langchain"]
+    elif flavor == "fastai":
+        import fastai
+        kwargs["pip_requirements"] = [f"mlflow=={latest_mlflow_version}", f"fastai=={fastai.__version__}"]
```

Comment:
The logged fastai model logs dependencies like:

```
astunparse==1.6.3
cffi==1.17.1
dill==0.3.8
fastai==2.7.17
google-cloud-storage==2.18.2
matplotlib==3.9.2
numpy==1.26.4
optree==0.13.0
packaging==24.1
pandas==2.2.3
protobuf==3.20.3
psutil==5.9.8
pyyaml==6.0.2
requests==2.32.3
scikit-learn==1.5.2
scipy==1.13.1
mlflow==2.16.2
```

and when installing these dependencies in Model serving container, it triggers pip dependency resolution and downloads many torch version packages and end up with "device no space left":

https://github.com/mlflow-automation/mlflow/actions/runs/11216475800/job/31175853638#step:11:1046

 Simplifying the dependency list can address this issue.